DMCompiler/Compiler/DMPreprocessor/DMPreprocessor.cs

```diff
@@ -165,7 +165,7 @@ public sealed class DMPreprocessor : IEnumerable<Token> {
                             yield return whitespace;
                         }
                         _currentLineContainsNonWhitespace = true;
-                        _canUseDirective = false;
+                        _canUseDirective = true;
```

Comment:
Why'd you switch this to true?

Reply:
Because its entirely valid in BYOND to do `var/thing = "stuff" #define ONE 1`

Reply:
I’m fairly sure that’s an error.

This change also leads to `_canUseDirectives` never being false.

Reply:
Ah, you are right, `_canUseDirective` should be set back to true if a `;` is encountered, but that's it